GradeCenter.API/Controllers/SchoolController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using GradeCenter.API.Models.Request;
+using GradeCenter.Services.Schools;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GradeCenter.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SchoolController : ControllerBase
+    {
+        private readonly ISchoolService _schoolService;
+
+        public SchoolController(ISchoolService schoolService)
+        {
+            _schoolService = schoolService;
+        }
+
+        /// <summary>
+        /// Creates an object of type School and stores it in the database.
+        /// </summary>
+        /// <param name="requestModel"></param>
+        /// <returns></returns>
+        [HttpPost("Create")]
+        public async Task<IActionResult> Create(SchoolCreateRequestModel requestModel)
+        {
+            await _schoolService.Create(requestModel.Name, requestModel.Address, requestModel.PrincipalId, requestModel.PrincipalFirstName, requestModel.PrincipalLastName);
+
+            return Ok();
+        }
+
+        /// <summary>
+        /// Updates an object of type School and stores it in the database.
+        /// </summary>
+        /// <param name="requestModel"></param>
+        /// <returns></returns>
+        [HttpPut("Update")]
+        public async Task<IActionResult> Update(SchoolUpdateRequestModel requestModel)
+        {
+            await _schoolService.Update(requestModel.Id, requestModel.Name, requestModel.Address);
+
```

Comment:
Here you can also pass the whole object

Reply:
The previously mentioned comment is also feasible here.

Reply:
> The previously mentioned comment is also feasible here.

same reply as for the Create method